core/src/test/java/com/linecorp/armeria/internal/common/DefaultHttpRequestTest.java

```diff
@@ -53,19 +54,19 @@
     void abortedAggregation(boolean executorSpecified, boolean withPooledObjects, Throwable abortCause) {
         final Thread mainThread = Thread.currentThread();
         final DefaultHttpRequest req = new DefaultHttpRequest(RequestHeaders.of(HttpMethod.GET, "/foo"));
-        final CompletableFuture<AggregatedHttpRequest> future;
+        final CompletableFuture<? extends AggregatedHttpRequest> future;
 
         // Practically same execution, but we need to test the both case due to code duplication.
         if (executorSpecified) {
             if (withPooledObjects) {
-                future = req.aggregateWithPooledObjects(
+                future = PooledHttpRequest.of(req).aggregateWithPooledObjects(
```

Comment:
Could use `aggregate()` instead of `aggregateWithPooledObjects`?

Reply:
Don't think so since this is `withPooledObjects` branch